app/src/fdroid/java/org/fossasia/openevent/app/common/di/module/android/ActivityBuildersModule.java

```diff
@@ -0,0 +1,62 @@
+package org.fossasia.openevent.app.common.di.module.android;
+
+import org.fossasia.openevent.app.core.auth.AuthActivity;
+import org.fossasia.openevent.app.core.event.about.AboutEventActivity;
+import org.fossasia.openevent.app.core.event.chart.ChartActivity;
+import org.fossasia.openevent.app.core.event.create.CreateEventActivity;
+import org.fossasia.openevent.app.core.event.create.UpdateEventFragment;
+import org.fossasia.openevent.app.core.event.create.EventDetailsStepOne;
+import org.fossasia.openevent.app.core.event.create.EventDetailsStepThree;
+import org.fossasia.openevent.app.core.event.create.EventDetailsStepTwo;
+import org.fossasia.openevent.app.core.main.MainActivity;
+import org.fossasia.openevent.app.core.organizer.detail.OrganizerDetailActivity;
+import org.fossasia.openevent.app.core.speaker.details.SpeakerDetailsActivity;
+import org.fossasia.openevent.app.core.speaker.details.SpeakerDetailsFragment;
+import org.fossasia.openevent.app.core.speakerscall.create.CreateSpeakersCallFragment;
+
+import dagger.Module;
+import dagger.android.ContributesAndroidInjector;
+
+@Module
+public abstract class ActivityBuildersModule {
```

Comment:
Bad, we are not going to maintain 30 files for each flavor, there should be ideally one file with as few lines as possible encapsulating the changes between flavors, take a look at OSS license PR